src/shared/components/KonvaVideo.tsx

```diff
@@ -33,13 +33,25 @@ export default function KonvaVideo({
     return element
   }, [src, loop])
 
-  // use Konva.Animation to redraw a layer
   useEffect(() => {
     if (!imageRef.current) return () => {}
 
-    videoElement.play()
+    // Source: https://stackoverflow.com/a/68128950
+    const handleVideoPLay = async () => {
+      const promise = videoElement.play()
+
+      if (promise !== undefined) {
+        promise.catch(() => {
```

Comment:
```suggestion
        promise?.catch(() => {
```

Reply:
We already check if promise is not `undefined`, no need for second check here